stratum/hal/bin/barefoot/README.md

```diff
@@ -233,6 +233,22 @@ journalctl -u stratum_bf.service
 
 See [gNMI CLI](/stratum/tools/gnmi/README.md)
 
+## P4DeviceConfig format and the DeviceConfigBuilder
+
+Stratum supports two different device configuration formats for pushing the P4
+pipeline over P4Runtime. The older binary packing used by [PI](https://github.com/p4lang/PI)
+and a newer more flexible protobuf based format ([bf.proto](stratum/hal/lib/barefoot/bf.proto)).
+You can use the device config builder to generate it:
+
+```bash
+bazel run //stratum/hal/bin/barefoot:device_config_builder -- \
+    -p4c_conf_file=/path/to/bf-p4c/compiler/output.conf \
+    -bfrt_device_config_binary_file=$PWD/device_config.pb.bin
```

Comment:
Isn't $PWD the default if we omit the directory?

Reply:
No, CWD will be a Bazel run directory: https://github.com/bazelbuild/bazel/issues/3325

Reply:
right. makes sense